src/Sentry.Unity.Native/SentryNative.cs

```diff
@@ -36,9 +36,6 @@ public static void Configure(SentryUnityOptions options)
 
                 //     return crashedLastRun.Value;
                 // };
-                // At this point Unity has taken the signal handler and will not invoke the original handler (Sentry)
-                // So we register our backend once more to make sure user-defined data is available in the crash report.
-                SentryNativeBridge.ReinstallBackend();
```

Comment:
This was added to resolve an issue where Unity would take the signal handler after Sentry had taken it and it would not call us after doing its thing. 
Is removing this really intended?